### DISCUSSION TITLE
Can anyone help me with my config (AMD 3950x CPU)

### DISCUSSION THREAD

**Discussion by huangsuli** [swar/Swar-Chia-Plot-Manager#948] [Q&A]
Can anyone help me with this setup:
CPU: AMD 3950x 16c 32t
RAM: 96GB
SSD:
1 1TB NVme (Samsung 980 pro)
2 2TB Nvme(Samsung 980 pro + Sabrent Rocket A Plus)

Configs:

name: 1TBSamsung
max_plots: 999
farmer_public_key:
pool_public_key:
temporary_directory: C:\Plotter
temporary2_directory: C:\Plotter
destination_directory: G:
size: 32
bitfield: true
threads: 3
buckets: 128
memory_buffer: 7000
max_concurrent: 3
max_concurrent_with_start_early: 3
stagger_minutes: 45
max_for_phase_1: 2
concurrency_start_early_phase: 1
concurrency_start_early_phase_delay: 0
temporary2_destination_sync: false

name: 2TBSamsung
max_plots: 999
farmer_public_key:
pool_public_key:
temporary_directory: Z:\Plotter
temporary2_directory: Z:\Plotter
destination_directory: G:
size: 32
bitfield: true
threads: 3
buckets: 128
memory_buffer: 7000
max_concurrent: 6
max_concurrent_with_start_early: 6
stagger_minutes: 30
max_for_phase_1: 4
concurrency_start_early_phase: 1
concurrency_start_early_phase_delay: 0
temporary2_destination_sync: false

name: 2TBSabrent
max_plots: 999
farmer_public_key:
pool_public_key:
temporary_directory: H:\Plotter
temporary2_directory: H:\Plotter
destination_directory: G:
size: 32
bitfield: true
threads: 3
buckets: 128
memory_buffer: 7000
max_concurrent: 6
max_concurrent_with_start_early: 6
stagger_minutes: 30
max_for_phase_1: 4
concurrency_start_early_phase: 1
concurrency_start_early_phase_delay: 0
temporary2_destination_sync: false

With this setup, the finishing time of one plot is around 15-17 hours... I don't know which part went wrong...

Thanks for the help.

**Comment by SoMoney** [comment]
Your components are good so it's probably disk i/o through your OS & Motherboard.
Could be your video card hogging your lanes or you don't have Bifurcation set up.
 Check your BIOS and make sure your PCIe16 bus is set to 4x4x4x4

**Reply by huangsuli** [reply]
@SoMoney Thanks man, I will try your recommendation.
